source/wp-content/themes/wporg-showcase-2022/functions.php

```diff
@@ -200,3 +201,21 @@ function modify_search_query( $query ) {
 
 	return $query;
 }
+
+/**
+ * Robots "noindex" rules for specific parts of the showcase site.
+ *
+ * @param bool $noindex
+ *
+ * @return bool
+ */
+function set_noindex( $noindex ) {
+	global $pagename;
+
+	if ( 'thanks' == $pagename ) {
```

Comment:
Should this use `===`?

Reply:
That's a good question. I imagine a strict comparison is always preferred. I'm surprised this isn't a standard rule. I'll update.